external-crates/move/crates/move-analyzer/src/analysis/parsing_analysis.rs

```diff
@@ -802,47 +803,3 @@ fn parsing_mod_ident_to_map_key(
     )
     .to_string()
 }
-
```

Comment:
This was just moved around without change so that it can be used in `completions.rs`